test/base/BaseReactor.t.sol

```diff
@@ -153,80 +159,65 @@ abstract contract BaseReactorTest is GasSnapshot, ReactorEvents, Test, DeployPer
             uint256 fillContractOutputBalanceStart
         ) = _checkpointBalances();
 
-        // TODO: expand to allow for custom callbackData in 3rd param
         vm.expectEmit(true, true, true, true, address(reactor));
         emit Fill(orderHash, address(fillContract), swapper, order.info.nonce);
         // execute order
-        _snapStart("ExecuteSingleValidation");
+        _snapStart("BaseExecuteSingleWithFee");
         fillContract.execute(signedOrder);
         snapEnd();
 
+        uint256 feeAmount = uint256(outputAmount) * feeBps / 10000;
         assertEq(tokenIn.balanceOf(address(swapper)), swapperInputBalanceStart - inputAmount);
         assertEq(tokenIn.balanceOf(address(fillContract)), fillContractInputBalanceStart + inputAmount);
         assertEq(tokenOut.balanceOf(address(swapper)), swapperOutputBalanceStart + outputAmount);
-        assertEq(tokenOut.balanceOf(address(fillContract)), fillContractOutputBalanceStart - outputAmount);
+        assertEq(tokenOut.balanceOf(address(fillContract)), fillContractOutputBalanceStart - outputAmount - feeAmount);
+        assertEq(tokenOut.balanceOf(address(feeRecipient)), feeAmount);
     }
 
-    /// @dev Execute with a failing validation contract
-    function testBaseExecuteValidationContractFail(uint128 inputAmount, uint128 outputAmount, uint256 deadline)
```

Comment:
I only see this one as a fuzz test.. did you not want to snapshot it?

Reply:
I think same with `testBaseExecuteInvalidReactor` and `testBaseExecuteDeadlinePassed` ? did you want snaps for those?

Reply:
See my other comment, I kept the fuzz tests but moved them to the bottom of the file so the diff looks weird

Reply:
and the fuzz tests have no snapshots, the concrete tests get the snapshots now

Reply:
yep I saw that. It looked like you moved some to fuzz and then also changed the non fuzz to be the snapshotting ones. Was just noting that these 3 tests didnt have non-fuzz snapshot versions.

Reply:
its testing a failure case, so dont think a snapshot is as useful